dbc2val/dbcfeederlib/dbcparser.py

```diff
@@ -28,9 +28,22 @@
 class DBCParser:
     def __init__(self, dbcfile: str, use_strict_parsing: bool = True):
 
-        # Read DBC file
-        log.info("Reading DBC file {}".format(dbcfile))
-        self.db = cantools.database.load_file(dbcfile, strict=use_strict_parsing)
+        self.db = None
+        first = True
+        found_names = set()
+        for name in dbcfile.split(","):
+            filename = name.strip()
+            if filename in found_names:
+                log.warn("The DBC file {} has already been read".format(filename))
+            else:
+                found_names.add(filename)
+            if first:
+                log.info("Reading DBC file {} as first file".format(filename))
+                self.db = cantools.database.load_file(filename, strict=use_strict_parsing)
+                first = False
+            else:
+                log.info("Adding definitions from {}".format(filename))
```

Comment:
here you need to check for found_names again, did a test and loading two files says it has loaded the same file but does it anyway after

Reply:
Yes, right now it just gives a warning, it anyway loads it the second time. I can change so it only loads it once